dask_ml/metrics/pairwise.py

```diff
@@ -130,7 +151,13 @@ def rbf_kernel(X, Y=None, gamma=None):
 
 
 @derived_from(metrics.pairwise)
-def polynomial_kernel(X, Y=None, degree=3, gamma=None, coef0=1):
+def polynomial_kernel(
+    X: ArrayLike,
+    Y: Optional[ArrayLike] = None,
+    degree: int = 3,
+    gamma: Optional[int] = None,
+    coef0: int = 1,
```

Comment:
```suggestion
    coef0: float = 1,
```

may need to change to 1.0?